src/main/java/de/tudarmstadt/lt/cw/global/CWGlobal.java

```diff
@@ -47,6 +48,9 @@ public static void main(String args[]) throws IOException {
 				.withDescription(
 						"max. number of similar words to process for a given word (size of word subgraph to be clustered)")
 				.isRequired().create("N"));
+		options.addOption(OptionBuilder.withArgName("cwOption").hasArg()
+				.withDescription("CW weighting option:  \"TOP\"(default), \"DIST_LOG\", \"DIST_NOLOG\"")
```

Comment:
The names need some comments: what TOP, DIST_LOG and DIST_NOLOG means. E.g. "top sums over the neighbourhood’s classes, dist downgrades the influence of a neighbouring node by its degree (number of edges of node)". You can even add link to the example: http://wortschatz.informatik.uni-leipzig.de/~cbiemann/software/CW.html#_Toc135565171